docs/spec/auth/vesting.md

```diff
@@ -1,7 +1,5 @@
 # Vesting
```

Comment:
Various grammatical errors throughout, needs a slow re-read.

Reply:
Really? Can you please point them out? I've updated this spec several times as well as having read it many times.